src/main/java/seedu/address/model/pet/Appointment.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.pet;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Pet's upcoming appointment in the address book.
+ * Guarantees: immutable; is always valid.
+ */
+public class Appointment {
+    public final String value;
```

Comment:
Maybe storing both of the fields `DateTime` and `Location` separately instead of together as a string would be better for abstraction?

Reply:
For the time being, I do not think it is necessary. I will take a look at this again while adding test cases.

Reply:
I was thinking it would be useful to separate them, since it would be used for the sort command in the future if we want to sort the pets by their appointment dates and times!

Reply:
Alright will work on this while adding test cases.